logger/logger.go

```diff
@@ -0,0 +1,61 @@
+package logger
+
+import (
+	"errors"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
+	graylog "gopkg.in/gemnasium/logrus-graylog-hook.v2"
+)
+
+const (
+	srctype = "API"
+	srcname = "Positioning filter"
+)
+
+// ErrBadLogDestination is the error returned when configuring failed becase of wrong destination
+var ErrBadLogDestination = errors.New("logger: bad destination for logger ")
+
+// NewLogger initialized logger according to configuration
+func NewLogger(lc *Config) error {
+	var err error
+	switch lc.Output {
+	case "Stdout":
+		lc.setLoggerToStdout()
+	case "File":
+		err = lc.setLoggerToFile()
+		if err != nil {
+			return err
+		}
+	case "Graylog":
+		lc.setLoggerToGraylog()
+	default:
+		err = ErrBadLogDestination
+		return err
+	}
+	return nil
+}
+
+// setLoggerToFile initialize logger for writing to file
+func (lc *Config) setLoggerToFile() error {
+	f, err := os.OpenFile(lc.FileName, os.O_APPEND|os.O_CREATE|os.O_RDWR, 0666)
```

Comment:
shouldn't you handle an error there, if any?))

Reply:
this method returns error in all cases, so it will be nil or error from this part of code